app/redux/reducers/SetsReducer.js

```diff
@@ -93,6 +97,27 @@ const createDefaultState = () => {
     };
 };
 
+// Set default metric
+
+const setDefaultMetric = (state, action) => {
+    let newWorkoutData = state.workoutData.slice(0);
+    let latestSet = newWorkoutData[newWorkoutData.length - 1];
+    let setIndex = newWorkoutData.findIndex( set => set.setID === action.setID );
+    let set = newWorkoutData[setIndex];
+
+    let changes = {};
+    
+    if (!set.exercise && !set.weight && !set.rpe && !set.tags && !set.reps && !set.tags && !set.videoFileURL) {
```

Comment:
either move this to its own function or add a comment above it explaining what this if statement is trying to do

Reply:
Done